pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -21,13 +22,16 @@ import (
 )
 
 const (
+	systemProjectName             = "System"
```

Comment:
I believe we already have these constants defined somewhere, most likely in the network policy code. Lets put them to some common package accessible by both network plicy and auth packages